openwpm/commands/profile_commands.py

```diff
@@ -19,69 +24,56 @@
 class DumpProfileCommand(BaseCommand):
     """
     dumps a browser profile currently stored in <browser_profile_folder> to
-    <tar_location> in which both folders are absolute paths.
+    <tar_path> in which both folders are absolute paths.
     """
 
-    def __init__(self, dump_folder, close_webdriver, compress):
-        self.dump_folder = dump_folder
+    def __init__(self, tar_path: Path, close_webdriver: bool, compress: bool) -> None:
+        self.tar_path = tar_path
         self.close_webdriver = close_webdriver
         self.compress = compress
+        raise ConfigError(
+            "BROWSER %i: Profile dumping is currently unsupported. "
+            "See: https://github.com/mozilla/OpenWPM/projects/2." % self.browser_id
+        )
```

Comment:
You're right we can't access the `browser_id` here and the `NotImplementedError` is more appropriate.
However I want to raise this error in `__init__` to make sure the object creation fails, since that usually happens in the main thread.
This way the user is immediately informed that this command won't run instead of having to wait until the browser tries to execute it.

Reply:
That makes sense. How about we just don't include the `browser_id` in the error message then?